native/src/routes/CategoriesContainer.tsx

```diff
@@ -71,16 +69,8 @@ const CategoriesContainer = ({ navigation, route }: CategoriesContainerProps): R
   const error =
     data?.categories && !category && previousLanguageCode === languageCode ? ErrorCode.PageNotFound : response.error
 
-  // Workaround clear cache on refresh if city content can't be loaded.
-  // TODO IGAPP-1231: Proper cache invalidation for version updates
-  const clearResourcesAndCache = useCallback(() => {
-    dataContainer.clearInMemoryCache()
-    dataContainer.clearOfflineCache().catch(reportError)
```

Comment:
This should not be called without good reason, at least not during every refresh.

Reply:
No, so far it was only called for switching between the cms after tapping landing 10 times to make sure we use the correct content/resources. In the landing page this is still kinda fine but in a city content page it breaks stuff bc it also deletes the metadata like last updates/last usages.